### PR TITLE
Remove temporary code for upgrade test

### PR DESCRIPTION
**Pull request by ysksuzuki**
Signed-off-by: Yusuke Suzuki <yusuke-suzuki@cybozu.co.jp>

### CODE REVIEW

**Comment by masa213f** [comment]
@ymmt2005 
Sorry, I made a trivial mistake.
I'll ask someone else to review it once the CI will pass.

File: bin/run-dctest-suite.sh [hunk 1]
```diff
@@ -15,10 +15,15 @@ GOPATH=\${HOME}/go
 export GOPATH
 PATH=/usr/local/go/bin:\${GOPATH}/bin:\${PATH}
 export PATH
+if [ "${SUITE_NAME}" = "upgrade" ]; then
+  MACHINES_FILE=/tmp/release/dctest/output/machines.yml
+else
+  MACHINES_FILE=\${GOPATH}/src/github.com/cybozu-go/neco/dctest/output/machines.yml
+fi
```
Comment (binoue) [inline bin/run-dctest-suite.sh:22]: I think this logic can move to `dctest/Makefile`. 
Could you explain why you decide `machines.yml` location here?

File: dctest/Makefile [hunk 2]
```diff
@@ -16,8 +17,9 @@ ifeq ($(MENU_ARG),)
 else
   MENU = $(MENU_ARG)
 endif
+MACHINES_FILE ?= $(abspath $(OUTPUT)/machines.yml)
```
Comment (llamerada-jp) [inline dctest/Makefile:20]: Do we need to use `?=`?
The other variables are set by using `=`.

Comment (masa213f) [reply dctest/Makefile:20]: This line sets the default value of `MACHINES_FILE`.
If `MACHINES_FILE` is specified as the command line argument(e.g. `make MACHINES_FILE=/path/to/machines.yaml`), this assignment will be ignored.

**Pull request merged**